### PR TITLE
mp-b2b bundled products

### PR DESCRIPTION
**Pull request by gorav-sharma-metacube**
## PR Description
CC-16119: mp-b2b bundled products

### CODE REVIEW

File: resources/environments/environments_api_mp_b2b.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "environment": "api_mp_b2b", 
 
     "env": {
-        "glue_url": "http://glue.de.spryker.local",
+        "glue_url": "https://glue.de.mp-b2b.internal-testing.demo-spryker.com",
```
Comment (OKrivtsova) [inline resources/environments/environments_api_mp_b2b.json:5]: please revert this change

**Review by OKrivtsova — APPROVED** [review]

File: resources/environments/environments_api_mp_b2b.json (hunk 1 above)
Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_mp_b2b.json]: Fixed

**Review by OKrivtsova — APPROVED** [review]

**Review by OKrivtsova — APPROVED** [review]
